lib/CompilerPluginSupport/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ if(SWIFT_SWIFT_PARSER)
   add_library("${library_name}" SHARED
     CompilerPluginSupport.swift)
 
+  set_target_properties(${library_name}
+    PROPERTIES
+      ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib/swift/host"
+      LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib/swift/host"
+  )
```

Comment:
Please add `RUNTIME_OUTPUT_DIRECTORY` as well.  I don't think that we can use `lib/swift/host` on Windows.  The general convention there is that the `RUNTIME_OUTPUT_DIRECTORY` is `bin` and the `LIBRARY_OUTPUT_DIRECTORY` and `ARCHIVE_OUTPUT_DIRECTORY` is `lib`.  That puts the import library (the moral equivalent of TBDs) and static libraries into `lib` and DLLs into `bin` where it will be found when doing the library lookup as there is no concept of RPATH.

Reply:
(I see how it's used elsewhere, never mind. I'll keep poking at this)